### PR TITLE
Feat/migrate up-download app to molgenis-components

### PR DESCRIPTION
**Pull request by chinook25**

### CODE REVIEW

File: apps/updownload/src/components/Import.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@
       <div v-if="taskUrl">
         <h4>Progress of current upload:</h4>
         <ul class="fa-ul">
-          <Task :task="task" />
+          <SubTask :task="task" />
```
Comment (mswertz) [inline apps/updownload/src/components/Import.vue:10]: Should be still 'Task' right?

Comment (MaxPostema) [reply apps/updownload/src/components/Import.vue:10]: There was a local component in this repo that was called task but did the same thing as the "SubTaks" component in the component-library. I opted for using the component from the library and removing the local one.

Comment (MaxPostema) [reply apps/updownload/src/components/Import.vue:10]: Now also removed the unused component. ;)

**Review by MaxPostema — DISMISSED** [review]
Upload of zip gives null error.

File: apps/tables/src/main.js [hunk 2]
```diff
@@ -3,6 +3,8 @@ import VueRouter from "vue-router";
 import ListTables from "./components/ListTables";
 import ViewTable from "./components/ViewTable";
 
+import "molgenis-components/dist/style.css";
```
Comment (connoratrug) [inline apps/tables/src/main.js:6]: other PR ?

**Review by connoratrug — APPROVED** [review]
- still get the console error after zip upload ( same as old version) 
- includes css import in tables app

**Pull request merged**